docs/spelling_wordlist.txt

```diff
@@ -1259,6 +1261,7 @@ sharded
 shellcheck
 shellcmd
 shm
+shouldn
```

Comment:
Where is this used?

Reply:
The spellchecker didn't like the contraction "shouldn't" (and "aren't either).
![image](https://user-images.githubusercontent.com/48934154/151839496-c5bb28a3-aa43-40fd-9e6e-ec17effbad81.png)

Reply:
Hmm, it’s strange this is showing up here; this PR is not even changing baseoperator.py.